examples/cloud-tomee-azure/README_pt.adoc

```diff
@@ -0,0 +1,111 @@
+= Implantação de TomEE no Azure
+:index-group: Cloud
+:jbake-type: page
+:jbake-status: published
+
+Este exemplo usa um aplicativo de eco básico, implantado com o TomEE incorporado na nuvem do Azure.
```

Comment:
I think we can call the app "echo" instead "eco" because it is the name of the application in English and also says the intent of the application by remembering the "echo" of the terminal.

Reply:
done. thanks.